installing/installing-using-helm.adoc

```diff
@@ -0,0 +1,27 @@
+[id="acs-installation"]
+= Installing {product-title} using Helm charts
+include::modules/common-attributes.adoc[]
+:context: acs-installation
+
+toc::[]
+
+{product-title} installs as a set services on your {ocp} cluster.
+
+High-level installation flow:
+
+. Add {product-title} Helm chart repository.
+. Configure the `central-services` Helm chart.
+. Install the xref:../architecture/index.adoc#centralized-components_acs-architecture[centralized components]  (Central and Scanner).
+. Install and set up the `roxctl` command-line interface.
+. Generate an init bundle.
+. Configure the `secured-cluster-services` Helm chart.
+. Install xref:../architecture/index.adoc#per-cluster-components_acs-architecture[per-cluster] and xref:../architecture/index.adoc#per-node-components_acs-architecture[per-node] components (Sensor, Admission Controller, and Collector).
+
+Before you install:
```

Comment:
Consider making this a "== Prerequisites" heading instead.

For example, like: https://docs.openshift.com/container-platform/4.7/installing/installing_aws/installing-aws-default.html

Reply:
It's not prerequisites, it's the installation flow, so users will need to do the following tasks to install RHACS. Each task is a procedure and it will appear as a heading on this page.